README.md

```diff
@@ -101,7 +101,9 @@ Done. This command should link `spaceship.zsh` as `prompt_spaceship_setup` to yo
 
 ### [prezto]
 
-`prezto` follows vanilla Zsh prompt setup. Just use [npm](#npm) or follow instructions in [manual](https://github.com/denysdovhan/spaceship-prompt#manual) installation.
+* Clone the [prezto-contrib](https://github.com/belak/prezto-contrib#usage) repo to the proper location.
+* Enable the `contrib-prompt` module (Before the `prompt` module).
```

Comment:
To me personally this would be a bit more clear:

* Follow the [prezto-contrib#usage](https://github.com/belak/prezto-contrib#usage) to clone `prezto-contrib` to the proper location.

Also, lowercase `B` in the `` Before the `prompt` ``

Reply:
Done. Thanks for the feedback. :)